bbc/simorgh
# WSTEAM1-931 - Spike - Article election banner

Part of https://jira.dev.bbc.co.uk/browse/WSTEAM1-931

Overall changes
======
(Hindi only for now)
- Adds dedicated component for rendering the Election banner on Article pages. The embed is rendered within an iframe to reduce page weight and also enable AMP support
- Adds a container that the iframe renders into to prevent excessive CLS
- Adds checks for toggle (implemented on Test env)
- Adds checks for `thingsLabel` to only render when the election tag is in the article
- Storybook link: https://wsteam1-931-article-election-banner--5d28eb3fe163f6002046d6fa.chromatic.com/?path=/story/pages-article-page--hindi-with-election-banner 

Testing
======
- Visit http://localhost:7080/hindi/articles/c03rl9735klo?renderer_env=live
- Confirm that the election banner appears
- Visit http://localhost:7080/hindi/articles/c3glj1nw7vlo?renderer_env=live
- Confirm that the election banner does not appear
- Visit the AMP version of http://localhost:7080/hindi/articles/c03rl9735klo.amp?renderer_env=live
- Confirm that the election banner appears

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
